src/scss/trumps/_utilities.scss

```diff
@@ -122,6 +122,38 @@
     }
 }
 
+//
+// Typographic Colour Helpers
+// ==========================================================================
+
+.u-color-text {
+    color: $color-text;
+}
+
+.u-color-text--hint {
+    color: $color-text--hint;
+}
+
+.u-color-text--success {
+    color: $color-text--success;
+}
+
+.u-color-text--danger {
+    color: $color-text--danger;
+}
+
+.u-color-text--warning {
+    color: $color-text--warning;
+}
+
+.u-color-headings {
```

Comment:
Definitely understand needing the classes above – with the headings ones, shouldn't they already have this colour defined if they are a heading?

There are some utility classes that make other elements look like headings, so may be best to just use those, otherwise it's kinda encouraging people to make headings that don't follow the design conventions (and just have the colour)

Reply:
yeah that's true! I'll remove those two.